client/blocks/reader-combined-card/post.jsx

```diff
@@ -71,8 +72,13 @@ class ReaderCombinedCardPost extends React.Component {
 
 	render() {
 		const { post, streamUrl, isDiscover, isSelected } = this.props;
-		const hasAuthorName = has( post, 'author.name' );
+		const isLoading = ! post || post._state === 'pending';
```

Comment:
I wondered about that - we only check for `pending` in full post view. I'll update both.